app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -63,6 +74,22 @@ class HomeFragment : Fragment() {
         settingsButton.setOnClickListener { v ->
             onSettingsPressed?.invoke()
         }
+
+        registerForContextMenu(view)
+    }
+
+    override fun onCreateContextMenu(menu: ContextMenu?, v: View?, menuInfo: ContextMenu.ContextMenuInfo?) {
+        super.onCreateContextMenu(menu, v, menuInfo)
+//        menu?.setHeaderTitle("Select Option")
+//        menu?.add(Menu.NONE, v!!.getId(), 0, "Remove tile")
```

Comment:
we don't need the word 'tile'

Reply:
also just want to double check that the string is from the string file so it gets translated.